.github/workflows/release-R.yml

```diff
@@ -21,10 +21,6 @@ jobs:
           enforce_admins: false
           branch: master
 
-      - name: Setup node
-        uses: actions/setup-node@v3
-        with:
-          node-version: '16'
```

Comment:
must be kept as we also running `npm version`

Reply:
No need of npm or node for R project.